### ISSUE TITLE
Generate and provide Paths_ modules during compilation

### ISSUE THREAD

**Issue by mrkkrp** [tweag/rules_haskell#137]
Many projects rely on existence of Cabal-generated `Path_<package_name>` modules which we currently don't emulate. This leads to compilation failures. We should generate such a module and provide it during compilation.

**Comment by Fuuzetsu** [comment]
I think solving this is quite high on priority list, lots of packages need this including many of my own.

**Comment by mboes** [comment]
Replacing Cabal is not the point of this project. It's not a high priority to be able to rebuild all of Hackage as-is. There are better ways of doing that already available, that we can and do reuse wholesale (Nixpkgs in particular).

**Comment by Fuuzetsu** [comment]
I don't think this is about replacing Cabal but making it not impossible for packages which rely on being able to find their data files to be built with rules_haskell. I'm happy for this to be solved without Paths_ but we shoud have some alternative.
